plugins/modules/ndo_physical_interface.py

```diff
@@ -0,0 +1,411 @@
+    - The old O(interfaces) will be replaced with the new O(interfaces) during an update.
+    type: list
+    elements: str
+  physical_interface_type:
+    description:
+    - The type of the interface policy group.
+    type: str
+    choices: [ physical, breakout ]
+  physical_policy_uuid:
+    description:
+    - The UUID of the Interface Setting Policy.
+    - This is only required when creating a new Port Channel Interface.
+    - This parameter is required when O(physical_interface_type) is C(physical).
+    - This parameter can be used instead of O(physical_policy).
+    type: str
+    aliases: [ policy_uuid, interface_policy_uuid , interface_policy_group_uuid, interface_setting_uuid]
+  physical_policy:
+    description:
+    - The interface group policy required for physical Interface Setting Policy.
+    - This parameter is required when O(physical_interface_type) is C(physical).
+    - This parameter can be used instead of O(physical_policy_uuid).
+    type: dict
+    suboptions:
+      name:
+        description:
+        - The name of the Port Channel Interface Setting Policy.
+        type: str
+      template:
+        description:
+        - The name of the template in which is referred the Port Channel Interface Policy Group.
+        type: str
+    aliases: [ policy, interface_policy, interface_policy_group, interface_setting ]
+  breakout_mode:
+    description:
+    - Breakout mode enables breaking down an ethernet port into multiple low-speed ports.
+    - This parameter is available only when O(physical_interface_type) is C(breakout).
+    - The default value is C(4x10G).
+    type: str
+    choices: [ 4x10G, 4x25G, 4x100G ]
+  interface_descriptions:
+    description:
+    - The interface settings defined in the interface settings policy will be applied to the interfaces on the node IDs configured in C(nodes).
+    type: list
+    elements: dict
+    suboptions:
+      interface_id:
+        description:
+        - The interface ID.
+        type: str
+      description:
+        description:
+        - The description of the interface.
+        type: str
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+notes:
+- The O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_template) to create the Tenant template.
+- The O(physical_policy) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_interface_setting) to create the Interface Setting Policy.
+- The O(physical_policy_uuid) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_interface_setting) to create the Interface Setting Policy UUID.
+seealso:
+- module: cisco.mso.ndo_template
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
```

Comment:
Please format the examples with appropriate spacing

Reply:
changed